account_info_query_unit_test.go

```diff
@@ -75,8 +75,6 @@ func TestUnitAccountInfoQueryGet(t *testing.T) {
 	balance.GetMaxBackoff()
 	balance.GetMaxRetryCount()
 	balance.GetPaymentTransactionID()
-	balance.GetQueryPayment()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why remove those two?

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏I see there are number of such tests that only invoke `balance.Get` methods without any assertions, meaning those tests provide no actual value and only increase the test coverage. Perhaps in a future PR we can review those and add assertions.